src/content/en/tools/lighthouse/index.md

```diff
@@ -0,0 +1,76 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Learn how to set up Lighthouse to audit your web apps.
+
+{# wf_updated_on: 2016-09-27 #}
+{# wf_published_on: 2016-09-27 #}
+
+# Audit Web Apps With Lighthouse {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Lighthouse is an open-source, automated tool for improving the quality of
+your web apps. You can run it as a Chrome Extension or from the command line.
+You give Lighthouse a URL that you want to audit, it runs a barrage of tests
+against the page, and then it generates a report on how well the page did.
+From here you can use the failing tests as indicators of what you can do to
+improve your app.
+
+## Get started
+
+There are two ways to run Lighthouse, as a Chrome Extension, or as a command
+line tool. The Chrome Extension provides a more user-friendly interface for
+reading reports. The command line tool enables you to integrate Lighthouse into
+continuous integration systems.
+
+### Chrome Extensions
+
+Download Google Chrome 52 or later.
```

Comment:
53 is stable now. I wonder if we need this or it could just be noted that you need 52 or greater.